test/utest/src/cuter_codeserver_tests.erl

```diff
@@ -5,17 +5,11 @@
 -include_lib("eunit/include/eunit.hrl").
 -include_lib("include/cuter_macros.hrl").
 
--spec test() -> 'ok' | {'error', term()}.  %% This should be provided by EUnit
+-spec test() -> ok | {error, any()}.  %% This should be provided by EUnit
 
 -define(MODS_LIST, [lists, dict, orddict, ets, os, string, filelib, beam_lib, cerl]).
 
--type descr() :: nonempty_string().
--type f_one() :: fun((term()) -> term()).
--type setup() :: {'setup', fun(() -> term()), f_one(), f_one()}.
--type ret_t() :: {descr(), setup()}.
-
-%% Load modules
--spec load_modules_test_() -> ret_t().
+-spec load_modules_test_() -> any().
```

Comment:
This function returns a 2-tuple, with a non-empty string in its first element, not `any()`...

Reply:
Added an include file and moved these typedefs there. Now we can reuse them for all utest modules.